interop/authzen-todo-backend/test/runner.ts

```diff
@@ -0,0 +1,98 @@
+import arrayToTable from "array-to-table";
+import clc from "cli-color";
+
+import { decisions } from "./decisions.json";
+
+const AUTHZEN_PDP_URL =
+  process.argv[2] || "https://authzen-proxy.demo.aserto.com";
+const AUTHZEN_PDP_API_KEY = process.env.AUTHZEN_PDP_API_KEY;
+
+const FORMAT = process.argv[3] === "markdown" ? "markdown" : "console";
+
+interface Result {
+  request: (typeof decisions)[number]["request"];
+  expected: (typeof decisions)[0]["expected"];
+  response?: boolean;
+  status: "PASS" | "FAIL" | "ERROR";
+  error?: string;
+}
+
+async function main() {
+  const results: Result[] = [];
+
+  for (const decision of decisions) {
+    const REQ = decision.request;
+    const EXP = decision.expected;
+    try {
+      const response = await fetch(`${AUTHZEN_PDP_URL}/access/v1/evaluation`, {
+        method: "POST",
+        headers: {
+          Authorization: AUTHZEN_PDP_API_KEY,
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(REQ),
+      });
+
+      const data = await response.json();
+      const RSP = data.decision || false;
+
+      results.push({
```

Comment:
Thanks @alexolivier ! Sorry for being so late with this review, just got a chance to look at it.

My only feedback is that for the `console` output, I think we should emit the result immediately - which will preserve the current behavior of giving the user immediate feedback as the tests are executed. 

For the `markdown` output, I think the current code makes perfect sense.

Reply:
Pushed an update to log results immediately when using console mode